### ISSUE TITLE
Mylar and SABNZBD Not Communicating

### ISSUE THREAD

**Issue by cwsleeper** [evilhero/mylar#1761]
I cannot get SABnzbd and Mylar to communicate any longer.

I've been using Mylar for at least 5 years and this is a bit weird.

Mylar indicates it is sending the requests to SABnzbd and marking the items as snatched. SABnzbd isn't registering any type of request. When I check the NZB key in Mylar, SAB gets that request, does the initial check on the API key and gives the standard warning. But that is it.

I am running both SAB and Mylar on the same machine. I have had issues in the past where SAB was stuck in fetch mode and generally a kick in the Mylar settings got things going again. I guess what I am trying to say is that while I am not a programmer, I have had much success in the past figuring out what may be wrong and getting things running again. Not this time. I am surprised I can't even get it to be stuck in fetch mode.

I can send you log files as needed. I am running the dev version of Mylar and the most recent version of SABnzbd. I've been trying off and on for a few weeks without success. Anything would help at this point. Thanks.

**Comment by evilhero** [comment]
Yeah it sounds like one of the settings is messed up as there haven't been any changes to the sab stuff in a while (there's something big coming, but that's another story).

If you could provide the debug logs showing the send-to-sab lines and then any errors in the logs it would help pinpoint things a bit more as I can't duplicate a problem at this point.

Also, make sure you're up to date on the development branch just by confirming the commit (located at the bottom of every page). There have been some issues with people updating properly where it's saying it's up-to-date, but it's really not.

**Comment by cwsleeper** [comment]
Thanks for the quick response.

I am running Version: 8673896d2f1ac69391c3d3c9bea021e565f2c85b (development)

I shut down and restarted from terminal. Then re-snatched an already snatched comic. I grabbed everything off terminal up to when the comic status is changed to Snatched.

02-Nov-2017 18:46:01 - INFO :: MAIN : Successfully discovered local IP and locking it in as : 10.9.0.10
02-Nov-2017 18:46:01 - INFO :: MAIN : Branch detected & set to : development
02-Nov-2017 18:46:01 - INFO :: MAIN : [DB Module] Loading : sqlite3 as the database module to use.
02-Nov-2017 18:46:01 - INFO :: MAIN : [Cache Check] Cache directory currently set to : /Applications/Usenet/Mylar/cache/
02-Nov-2017 18:46:01 - INFO :: MAIN : Checking to see if the database has all tables....
02-Nov-2017 18:46:02 - INFO :: MAIN : Populating Base Exception listings into Mylar....
02-Nov-2017 18:46:02 - INFO :: MAIN : Populating Custom Exception listings into Mylar....
02-Nov-2017 18:46:02 - INFO :: MAIN : Ensuring DB integrity - Removing all Erroneous Comics (ie. named None)
02-Nov-2017 18:46:02 - INFO :: MAIN : Correcting Null entries that make the main page break on startup.
02-Nov-2017 18:46:02 - INFO :: MAIN : Background queuing up a refresh of 0 series' that contain issues that have a None status and shouldn't
02-Nov-2017 18:46:02 - INFO :: MAIN : job_history loaded: [(u'DB Updater', u'2017-11-02 22:39:30.670328', 1509662370.670328, u'2017-11-02 22:44:30.670664', 1509662670.670664, u'True', None, None, u'Waiting'), (u'Check Version', u'2017-11-02 11:57:46.171777', 1509623866.171777, u'2017-11-03 02:59:30', 1509677970.0, u'True', None, None, u'Waiting'), (u'Auto-Search', u'2017-11-02 11:30:22.194542', 1509622222.194542, u'2017-11-03 02:59:30', 1509677970.0, u'True', None, None, u'Waiting'), (u'Weekly Pullist', u'2017-11-02 12:02:46.400650', 1509624166.40065, u'2017-11-03 12:02:46', 1509710566.0, u'True', None, None, u'Waiting')]
02-Nov-2017 18:46:02 - INFO :: MAIN : Retrieving latest version information from github
02-Nov-2017 18:46:02 - INFO :: MAIN : Mylar is up to date
02-Nov-2017 18:46:02 - INFO :: MAIN : Remapping the sorting to allow for new additions.
02-Nov-2017 18:46:02 - INFO :: MAIN : Sucessfully ordered 2733 series in your watchlist.
02-Nov-2017 18:46:02 - INFO :: MAIN : Starting Mylar on http://0.0.0.0:8090/
02-Nov-2017 18:46:03 - INFO :: MAIN : [WEEKLY] Checking for existance of Weekly Comic listing...
02-Nov-2017 18:46:03 - INFO :: MAIN : Firing up the Background Schedulers now....
Pending jobs:
    Status Updater (trigger: date[2017-11-02 18:46:02 EDT], pending)
    DB Updater (trigger: interval[0:05:00], pending)
    Auto-Search (trigger: interval[6:00:00], pending)
    Weekly Pullist (trigger: interval[1 day, 0:00:00], next run at: 2017-11-03 12:02:46 UTC)
    Check Version (trigger: interval[6:00:00], pending)
02-Nov-2017 18:46:03 - INFO :: MAIN : Background Schedulers successfully started...
02-Nov-2017 18:46:24 - INFO :: Thread-13 : Issue Title given as : the body in the library part 2 of 2
02-Nov-2017 18:46:24 - INFO :: Thread-13 : search provider order is ['newznab:http://localhost']
02-Nov-2017 18:46:24 - INFO :: Thread-13 : Shhh be very quiet...I'm looking for Usagi Yojimbo issue: 162 (2017) using http://localhost (newznab)
02-Nov-2017 18:46:24 - INFO :: Thread-13 : pausing for 60 seconds before continuing to avoid hammering
02-Nov-2017 18:47:24 - INFO :: Thread-13 : no errors on data retrieval...proceeding
02-Nov-2017 18:47:24 - INFO :: Thread-13 : entry: {'newznab': {u'size': u'88180543', 'id': u'89d0a0e3a55629a650748ed40830bae7', 'tags': {}}, 'updated': u'Sat, 28 Oct 2017 16:08:11 -0400', 'updated_parsed': time.struct_time(tm_year=2017, tm_mon=10, tm_mday=28, tm_hour=20, tm_min=8, tm_sec=11, tm_wday=5, tm_yday=301, tm_isdst=0), 'links': [{'href': u'http://localhost/getnzb/89d0a0e3a55629a650748ed40830bae7.nzb&i=1&r=42faf956fe6e9828ee3096a4d9209db5', 'type': 'text/html', 'rel': 'alternate'}, {'length': u'88180543', 'href': u'http://localhost/getnzb/89d0a0e3a55629a650748ed40830bae7.nzb&i=1&r=42faf956fe6e9828ee3096a4d9209db5', 'type': u'application/x-nzb', 'rel': 'enclosure'}], 'title': u'Usagi Yojimbo 162 (2017) (digital) (Son of Ultron-Empire)', 'tags': [{'term': u'Books > Comics', 'scheme': None, 'label': None}], 'summary_detail': {'base': u'', 'type': 'text/html', 'value': u'Usagi Yojimbo 162 (2017) (digital) (Son of Ultron-Empire)', 'language': None}, 'comments': u'http://localhost/details/89d0a0e3a55629a650748ed40830bae7#comments', 'summary': u'Usagi Yojimbo 162 (2017) (digital) (Son of Ultron-Empire)', 'guidislink': True, 'title_detail': {'base': u'', 'type': 'text/plain', 'value': u'Usagi Yojimbo 162 (2017) (digital) (Son of Ultron-Empire)', 'language': None}, 'link': u'http://localhost/getnzb/89d0a0e3a55629a650748ed40830bae7.nzb&i=1&r=42faf956fe6e9828ee3096a4d9209db5', 'id': u'http://localhost/details/89d0a0e3a55629a650748ed40830bae7'}
02-Nov-2017 18:47:24 - INFO :: Thread-13 : Found Usagi Yojimbo (2017) issue: 162 using http://localhost (newznab)
02-Nov-2017 18:47:24 - INFO :: Thread-13 : Download URL: http://localhost/api?apikey=42faf956fe6e9828ee3096a4d9209db5&t=get&id=89d0a0e3a55629a650748ed40830bae7 [VerifySSL:True]
02-Nov-2017 18:47:25 - INFO :: Thread-13 : Successfully sent nzb file to SABnzbd
02-Nov-2017 18:47:25 - INFO :: Thread-13 : setting the nzbid for this download grabbed by http://localhost (newznab) in the nzblog to : 89d0a0e3a55629a650748ed40830bae7
02-Nov-2017 18:47:25 - INFO :: Thread-13 : [UPDATER] Updating status to snatched
02-Nov-2017 18:47:25 - INFO :: Thread-13 : [UPDATER] Updated the status (Snatched) complete for Usagi Yojimbo Issue: 162

**Comment by evilhero** [comment]
It looks like you have verifySSL set to true for your sabnzbd within mylar, but your only hitting the http port.

If it's a non-https instance of sab, turn off the verifyssl for sab and save the config, then restart it so the changes will load properly. Otherwise if it is an https sab instance, change the http to https and do the save, restart.

That might do the trick...

**Comment by cwsleeper** [comment]
Thanks. I saw that when I uploaded the log, but knew I had nothing set for SSL. I checked the config.ini and I still don't. As far as I can tell, there is no entry under the SAB settings for SSL. The SAB host is http. 
I checked the Verify SSL setting under Newznab and it was set to 0.

extra_newznabs = http://localhost, http://localhost, 0, 42faf956fe6e9828ee3096a4d9209db5, "", 1

So, I stopped Mylar and re-started and did another comic. The same line [VerifySSL:True] shows up.

I there an SAB setting I don't see? The sad_to_mylar is the separate machines toggle, I thought.

Should I have an SSL setting in here?

[SABnzbd]
sab_host = http://localhost:8080
sab_username = 
sab_password =
sab_apikey = 3dd4e07d91a42fffcdbc4e76c24cddc4
sab_category = comics
sab_priority = Default
sab_to_mylar = 0
sab_directory = ""

**Comment by evilhero** [comment]
Yeah that looks fine - I think we  need better logs. 

If you can enable the verbose logging, and then try a snatch and paste in the same sequence as you did above it should fill in some gaps that I need to see in order to nail down what's happening.

**Comment by cwsleeper** [comment]
02-Nov-2017 23:09:59 - INFO    :: MAIN : Successfully discovered local IP and locking it in as : 10.9.0.10
02-Nov-2017 23:09:59 - INFO    :: MAIN : Branch detected & set to : development
02-Nov-2017 23:09:59 - INFO    :: MAIN : [DB Module] Loading : sqlite3 as the database module to use.
02-Nov-2017 23:09:59 - INFO    :: MAIN : [Cache Check] Cache directory currently set to : /Applications/Usenet/Mylar/cache/
02-Nov-2017 23:09:59 - INFO    :: MAIN : Checking to see if the database has all tables....
02-Nov-2017 23:09:59 - INFO    :: MAIN : Populating Base Exception listings into Mylar....
02-Nov-2017 23:09:59 - INFO    :: MAIN : Populating Custom Exception listings into Mylar....
02-Nov-2017 23:09:59 - INFO    :: MAIN : Ensuring DB integrity - Removing all Erroneous Comics (ie. named None)
02-Nov-2017 23:09:59 - INFO    :: MAIN : Correcting Null entries that make the main page break on startup.
02-Nov-2017 23:09:59 - INFO    :: MAIN : Background queuing up a refresh of 0 series' that contain issues that have a None status and shouldn't
02-Nov-2017 23:09:59 - INFO    :: MAIN : job_history loaded: [(u'DB Updater', u'2017-11-03 03:09:16.845695', 1509678556.845695, u'2017-11-03 03:14:16.845941', 1509678856.845941, u'True', None, None, u'Waiting'), (u'Check Version', u'2017-11-02 11:57:46.171777', 1509623866.171777, u'2017-11-03 07:18:41', 1509693521.0, u'True', None, None, u'Waiting'), (u'Auto-Search', u'2017-11-02 11:30:22.194542', 1509622222.194542, u'2017-11-03 07:18:41', 1509693521.0, u'True', None, None, u'Waiting'), (u'Weekly Pullist', u'2017-11-02 12:02:46.400650', 1509624166.40065, u'2017-11-03 12:02:46', 1509710566.0, u'True', None, None, u'Waiting'), (u'RSS Feeds', None, None, u'2017-11-03 01:34:21', 1509672861.0, None, None, None, u'Waiting')]
02-Nov-2017 23:09:59 - INFO    :: MAIN : Retrieving latest version information from github
02-Nov-2017 23:09:59 - INFO    :: MAIN : Mylar is up to date
02-Nov-2017 23:09:59 - INFO    :: MAIN : Remapping the sorting to allow for new additions.
02-Nov-2017 23:09:59 - INFO    :: MAIN : Sucessfully ordered 2733 series in your watchlist.
02-Nov-2017 23:09:59 - INFO    :: MAIN : Starting Mylar on http://0.0.0.0:8090/
02-Nov-2017 23:10:00 - INFO    :: MAIN : [WEEKLY] Checking for existance of Weekly Comic listing...
02-Nov-2017 23:10:00 - INFO    :: MAIN : Firing up the Background Schedulers now....
02-Nov-2017 23:10:00 - INFO    :: MAIN : Background Schedulers successfully started...
02-Nov-2017 23:10:21 - INFO    :: CP Server Thread-7 : Verbose toggled, set to True
02-Nov-2017 23:10:21 - DEBUG   :: CP Server Thread-7 : If you read this message, debug logging is available
02-Nov-2017 23:10:43 - DEBUG   :: CP Server Thread-9 : Marking Stray Bullets: Sunshine & Roses 28 as Wanted
02-Nov-2017 23:10:43 - DEBUG   :: CP Server Thread-9 : updated...to Wanted
02-Nov-2017 23:10:43 - DEBUG   :: CP Server Thread-9 : Marking issues: ['626227'] as Wanted
02-Nov-2017 23:10:43 - INFO    :: Thread-13 : [SEARCH] Special Characters exist within Series Title. Enabling search-safe Name : Stray Bullets Sunshine & Roses
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : Publisher is : Image
02-Nov-2017 23:10:43 - INFO    :: Thread-13 : Issue Title given as : magic banana
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : Checking for torrent enabled.
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : newznab name:http://localhost @ http://localhost
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : nzbprovider(s): ['newznab:http://localhost']
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : there are : 0 torrent providers you have selected.
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : there are : 1 nzb providers you have selected.
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : Usenet Retention : 1500 days
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : found provider in existing enabled providers.
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : checking against nzb provider: newznab:http://localhost
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : newznab match against: newznab:http://localhost
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : sucessfully matched - appending to provider.order sequence
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : provider order sequence is now to start with http://localhost at spot #0
02-Nov-2017 23:10:43 - INFO    :: Thread-13 : search provider order is ['newznab:http://localhost']
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : Initiating Search via : api
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : using Newznab host of : http://localhost
02-Nov-2017 23:10:43 - INFO    :: Thread-13 : Shhh be very quiet...I'm looking for Stray Bullets: Sunshine & Roses issue: 28 (2017) using http://localhost (newznab)
02-Nov-2017 23:10:43 - DEBUG   :: Thread-13 : calpha/cnumber: None / None
02-Nov-2017 23:10:43 - INFO    :: Thread-13 : pausing for 60 seconds before continuing to avoid hammering
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : [SSL: False] Search URL: http://localhost/api?t=search&q=stray%20bullets%20sunshine%20roses%20028&o=xml&cat=7030&apikey=xUDONTNEEDTOKNOWTHISx
02-Nov-2017 23:11:43 - INFO    :: Thread-13 : no errors on data retrieval...proceeding
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : checking search result: Stray Bullets - Sunshine & Roses 028 (2017) (digital) (Son of Ultron-Empire)
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : sub:Stray Bullets - Sunshine & Roses 028 (2017) (digital) (Son of Ultron-Empire)
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : size given as: 83.3 MB
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : Sat, 28 Oct 2017 16:08:11 -0400 is after store date of 2017-10-04
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : Entry: Stray Bullets - Sunshine & Roses 028 (2017) (digital) (Son of Ultron-Empire)
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : Cleantitle: stray bullets - sunshine and roses 028 (2017) (digital) (son of ultron-empire)
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : Cleantitle: stray bullets - sunshine and roses 028 (2017) (digital) (son of ultron-empire)
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : 0. Bracket Word: stray bullets - sunshine and roses 028 
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : Comic: stray bullets - sunshine and roses 028 
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : UseFuzzy is  : 0
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : ComVersChk : 3
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : 1. Bracket Word: 2017
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : year detected: 2017
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : year looking for: 2017
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : 2017 - right years match baby!
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : 3. Bracket Word: digital
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : digital edition detected
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : 5. Bracket Word: son of ultron-empire
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : mini-series detected : son of ultron-empire
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : Scanner detected: son of ultron-empire
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : original nzb comic and issue: stray bullets - sunshine and roses 028 
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : [Stray Bullets: Sunshine & Roses] I have found a -  within the nzbname @ position: 14
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : There is no hyphen present in the series title.
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : Assuming position start is : 14
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : First word of issue stored as : sunshine
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : adjusted nzb comic and issue: stray bullets   sunshine and roses 028 
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : adjusting from: stray bullets   sunshine and roses 028  to: 028
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : ['stray', 'bullets', 'sunshine', 'and', 'roses', '028'] nzb series word count: 5
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : ['stray', 'bullets', 'sunshine', 'and', 'roses'] watchlist word count: 5
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : splitst : 5
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : len-watchcomic : 5
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : length match..proceeding
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : search-length: 5
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : Watchlist-length: 5
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : splitit: stray
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : scount : 0
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : 0 Comparing: stray .to. stray
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : word matched on : stray
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : splitit: bullets
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : scount : 1
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : 1 Comparing: bullets .to. bullets
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : word matched on : bullets
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : splitit: sunshine
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : scount : 2
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : 2 Comparing: sunshine .to. sunshine
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : word matched on : sunshine
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : splitit: and
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : scount : 3
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : 3 Comparing: and .to. and
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : word matched on : and
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : splitit: roses
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : scount : 4
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : 4 Comparing: roses .to. roses
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : word matched on : roses
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : scount:5
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : splitit-len:5
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : Match to series : 100.0 %.
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : it's a go captain... - we matched 100.0%!
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : this should be a match!
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : issue we are looking for is : 28
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : integer value of issue we are looking for : 28000
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : issue we found for is : 028
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : integer value of issue we have found : 28000
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : [SEARCHER] entry[title]: Stray Bullets - Sunshine & Roses 028 (2017) (digital) (Son of Ultron-Empire)
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : [SEARCHER] nzbname (remove chars): Stray Bullets - Sunshine and Roses 028  2017   digital   Son of Ultron-Empire 
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : [SEARCHER] nzbname (\s): Stray Bullets - Sunshine and Roses 028 2017 digital Son of Ultron-Empire
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : [SEARCHER] end nzbname: Stray.Bullets.-.Sunshine.and.Roses.028.2017.digital.Son.of.Ultron-Empire
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : nzbname used for post-processing:Stray.Bullets.-.Sunshine.and.Roses.028.2017.digital.Son.of.Ultron-Empire
02-Nov-2017 23:11:43 - INFO    :: Thread-13 : entry: {'newznab': {u'size': u'87387826', 'id': u'1d5b20e529b08fbbdefd560878782f67', 'tags': {}}, 'updated': u'Sat, 28 Oct 2017 16:08:11 -0400', 'updated_parsed': time.struct_time(tm_year=2017, tm_mon=10, tm_mday=28, tm_hour=20, tm_min=8, tm_sec=11, tm_wday=5, tm_yday=301, tm_isdst=0), 'links': [{'href': u'http://localhost/getnzb/1d5b20e529b08fbbdefd560878782f67.nzb&i=1&r=42faf956fe6e9828ee3096a4d9209db5', 'type': 'text/html', 'rel': 'alternate'}, {'length': u'87387826', 'href': u'http://localhost/getnzb/1d5b20e529b08fbbdefd560878782f67.nzb&i=1&r=42faf956fe6e9828ee3096a4d9209db5', 'type': u'application/x-nzb', 'rel': 'enclosure'}], 'title': u'Stray Bullets - Sunshine & Roses 028 (2017) (digital) (Son of Ultron-Empire)', 'tags': [{'term': u'Books > Comics', 'scheme': None, 'label': None}], 'summary_detail': {'base': u'', 'type': 'text/html', 'value': u'Stray Bullets - Sunshine & Roses 028 (2017) (digital) (Son of Ultron-Empire)', 'language': None}, 'comments': u'http://localhost/details/1d5b20e529b08fbbdefd560878782f67#comments', 'summary': u'Stray Bullets - Sunshine & Roses 028 (2017) (digital) (Son of Ultron-Empire)', 'guidislink': True, 'title_detail': {'base': u'', 'type': 'text/plain', 'value': u'Stray Bullets - Sunshine & Roses 028 (2017) (digital) (Son of Ultron-Empire)', 'language': None}, 'link': u'http://localhost/getnzb/1d5b20e529b08fbbdefd560878782f67.nzb&i=1&r=42faf956fe6e9828ee3096a4d9209db5', 'id': u'http://localhost/details/1d5b20e529b08fbbdefd560878782f67'}
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : issues match!
02-Nov-2017 23:11:43 - INFO    :: Thread-13 : Found Stray Bullets: Sunshine & Roses (2017) issue: 28 using http://localhost (newznab)
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : link given by: newznab
02-Nov-2017 23:11:43 - INFO    :: Thread-13 : Download URL: http://localhost/api?apikey=42faf956fe6e9828ee3096a4d9209db5&t=get&id=1d5b20e529b08fbbdefd560878782f67 [VerifySSL:True]
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : [FILENAME] filename (remove chars): Stray Bullets - Sunshine and Roses 028  2017   digital   Son of Ultron-Empire 
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : [FILENAME] nzbname (\s): Stray Bullets - Sunshine and Roses 028 2017 digital Son of Ultron-Empire
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : [FILENAME] end nzbname: Stray.Bullets.-.Sunshine.and.Roses.028.2017.digital.Son.of.Ultron-Empire
02-Nov-2017 23:11:43 - DEBUG   :: Thread-13 : Cache Directory successfully found at : /Applications/Usenet/Mylar/cache/. Ensuring proper permissions.
02-Nov-2017 23:11:44 - DEBUG   :: Thread-13 : Successfully changed permissions [0777 / 0660]
02-Nov-2017 23:11:44 - DEBUG   :: Thread-13 : send-to-SAB host &api initiation string : http://localhost:8080/api?apikey=xUDONTNEEDTOKNOWTHISx
02-Nov-2017 23:11:44 - DEBUG   :: Thread-13 : ...selecting API type: http://localhost:8080/api?apikey=3dd4e07d91a42fffcdbc4e76c24cddc4&mode=addurl&name=
02-Nov-2017 23:11:44 - DEBUG   :: Thread-13 : ...attaching nzb via internal Mylar API: http://localhost:8080/api?apikey=3dd4e07d91a42fffcdbc4e76c24cddc4&mode=addurl&name=10.9.0.6%3A8090%2Fapi%3Fapikey%3D731fc6e56d439da3405eb0ec3bb92ebb%26cmd%3DdownloadNZB%26nzbname%3DStray.Bullets.-.Sunshine.and.Roses.028.2017.digital.Son.of.Ultron-Empire.nzb
02-Nov-2017 23:11:44 - DEBUG   :: Thread-13 : ...setting priority: http://localhost:8080/api?apikey=3dd4e07d91a42fffcdbc4e76c24cddc4&mode=addurl&name=10.9.0.6%3A8090%2Fapi%3Fapikey%3D731fc6e56d439da3405eb0ec3bb92ebb%26cmd%3DdownloadNZB%26nzbname%3DStray.Bullets.-.Sunshine.and.Roses.028.2017.digital.Son.of.Ultron-Empire.nzb&priority=-100
02-Nov-2017 23:11:44 - DEBUG   :: Thread-13 : ...attaching category: http://localhost:8080/api?apikey=3dd4e07d91a42fffcdbc4e76c24cddc4&mode=addurl&name=10.9.0.6%3A8090%2Fapi%3Fapikey%3D731fc6e56d439da3405eb0ec3bb92ebb%26cmd%3DdownloadNZB%26nzbname%3DStray.Bullets.-.Sunshine.and.Roses.028.2017.digital.Son.of.Ultron-Empire.nzb&priority=-100&cat=comics
02-Nov-2017 23:11:44 - DEBUG   :: Thread-13 : ...attaching rename script: http://localhost:8080/api?apikey=3dd4e07d91a42fffcdbc4e76c24cddc4&mode=addurl&name=10.9.0.6%3A8090%2Fapi%3Fapikey%3D731fc6e56d439da3405eb0ec3bb92ebb%26cmd%3DdownloadNZB%26nzbname%3DStray.Bullets.-.Sunshine.and.Roses.028.2017.digital.Son.of.Ultron-Empire.nzb&priority=-100&cat=comics&script=ComicRN.py
02-Nov-2017 23:11:44 - DEBUG   :: Thread-13 : Completed send-to-SAB link: http://localhost:8080/api?apikey=3dd4e07d91a42fffcdbc4e76c24cddc4&mode=addurl&name=10.9.0.6%3A8090%2Fapi%3Fapikey%3D731fc6e56d439da3405eb0ec3bb92ebb%26cmd%3DdownloadNZB%26nzbname%3DStray.Bullets.-.Sunshine.and.Roses.028.2017.digital.Son.of.Ultron-Empire.nzb&priority=-100&cat=comics&script=ComicRN.py
02-Nov-2017 23:11:45 - INFO    :: Thread-13 : Successfully sent nzb file to SABnzbd
02-Nov-2017 23:11:45 - DEBUG   :: Thread-13 : we should break out now - sucessful search previous
02-Nov-2017 23:11:45 - DEBUG   :: Thread-13 : Found matching comic...preparing to send to Updater with IssueID: 626227 and nzbname: Stray.Bullets.-.Sunshine.and.Roses.028.2017.digital.Son.of.Ultron-Empire
02-Nov-2017 23:11:45 - INFO    :: Thread-13 : setting the nzbid for this download grabbed by http://localhost (newznab) in the nzblog to : 1d5b20e529b08fbbdefd560878782f67
02-Nov-2017 23:11:45 - DEBUG   :: Thread-13 : Deleted stale entry from nzblog for IssueID: 626227 [http://localhost (newznab)]
02-Nov-2017 23:11:45 - DEBUG   :: Thread-13 : findit = found!
02-Nov-2017 23:11:45 - DEBUG   :: Thread-13 : [UPDATER] comicid: 79879
02-Nov-2017 23:11:45 - DEBUG   :: Thread-13 : [UPDATER] issueid: 626227
02-Nov-2017 23:11:45 - INFO    :: Thread-13 : [UPDATER] Updating status to snatched
02-Nov-2017 23:11:45 - DEBUG   :: Thread-13 : [UPDATER] Provider is newznab
02-Nov-2017 23:11:45 - INFO    :: Thread-13 : [UPDATER] Updated the status (Snatched) complete for Stray Bullets: Sunshine & Roses Issue: 28

**Comment by evilhero** [comment]
Perfect.

It looks like the url requesting the nzb is using a different ip than what it says at startup (10.9.0.6 vs 10.9.0.10), plus it's missing the http/https for the mylar host address in the addurl portion of the send-to-sab line.

Normally this discrepancy happens when the host_return value in the config.ini is set, as mylar will use that complete value but instead its using just the ip. So what I'm thinking is you have a host_return value set to ```10.9.0.6:8090```, in which case you could just blank out the value entirely, save the config and start mylar up and it should start working.

**Comment by cwsleeper** [comment]
Thanks. I was down for a few days replacing my HDD with an SSD then I was away so I couldn't check.

I removed that entry from the config.ini file - I expect I put that in there while trying to get this running again to begin with. It did not work at first, but Newznab was having an issue from replacing the HDD. 

Anyway, I rebooted and it is working as of this morning.

I did notice that the ip went back to a 192.168. ip. I don't know where the 10.9 ip started being assigned. I am not familiar with how ip assignments work, but I thought they were usually based on 192.168 or 10.0 

Regardless, it is working.

Lastly, thank you so much. I really appreciate not only the time you gave for my issue, but the work you do and time you put into this for everyone's issues. It reminded me that while I have "paid" for Mylar in the past, that continued development and great support still cost. As such, I went in and "paid" for an "upgrade". I honestly should set a reminder to do that yearly.
